gpbft/gpbft.go

```diff
@@ -552,7 +552,7 @@ func (i *instance) beginConverge() {
 			panic("beginConverge called but no justification for proposal")
 		}
 	}
-	_, pubkey := i.powerTable.Get(i.participant.id)
+	_, pubkey := i.powerTable.Get(i.participant.host.ID())
```

Comment:
I believe the idea is to move this into a host-based `broadcast` function, and let the client compute the tickets and signatures.